### PR TITLE
Overriding supported localization

### PR DESCRIPTION
**Pull request by vidorteg**
This PR overrides the list of supported locales on the Devtools as extension currently only support en-US

### CODE REVIEW

File: postBuildStep.ts [hunk 1]
```diff
@@ -177,6 +178,10 @@ async function patchFilesForWebView(toolsOutDir: string) {
     await patchFileForWebViewWrapper("sdk/sdk.js", toolsOutDir, [
         applyRerouteConsoleMessagePatch,
     ]);
+
```
Comment (mliao95) [inline postBuildStep.ts:181]: We can remove this newline

File: src/host/polyfills/thirdPartyI18n.test.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { testPatch } from "../../test/helpers";
+import applyThirdPartyI18nLocalesPatch from "./thirdPartyI18n";
+
+describe("override i18n locales", () => {
+    it("applySetupTextSelectionPatch correctly changes text", async () => {
```
Comment (mliao95) [inline src/host/polyfills/thirdPartyI18n.test.ts:7]: replace with applySetupTextSelectionPatch with applyThirdPartyI18nLocalesPatch

File: src/host/polyfills/thirdPartyI18n.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export default function applyThirdPartyI18nLocalesPatch(content: string) {
+    const pattern = /const\s*locales\s*=\s*\{[^]+?};/g;
```
Comment (mliao95) [inline src/host/polyfills/thirdPartyI18n.ts:4]: Let´s leave a comment here so we remember to remove this patch when we implement localization.

**Review by mliao95 — APPROVED** [review]
LGTM with some comments

File: src/host/polyfills/thirdPartyI18n.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export default function applyThirdPartyI18nLocalesPatch(content: string) {
```
[the remark this answers is not in the document]
Comment (vidorteg) [reply src/host/polyfills/thirdPartyI18n.ts:3]: as the rules are disabled ATM I think is fine to continue using the existing patterns but I agree this need to change we enable the linting.

File: src/host/polyfills/thirdPartyI18n.ts (hunk 3 above)
Comment (vidorteg) [reply src/host/polyfills/thirdPartyI18n.ts:7]: done.

**Pull request merged**